README.md

```diff
@@ -11,14 +11,20 @@
   <br />
   <br />
   <a href="https://circleci.com/gh/FormidableLabs/urql">
-    <img alt="Test Status" src="https://circleci.com/gh/FormidableLabs/urql.svg?style=shield" />
-  </a> <a href="https://github.com/FormidableLabs/urql#maintenance-status">
-    <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
+    <img alt="Test Status" src="https://badgen.net/github/status/FormidableLabs/urql/master/ci/circleci?label=circleci" />
+  </a>
+  <a href="https://www.npmjs.com/package/urql">
+    <img alt="Total downloads" src="https://badgen.net/npm/dt/urql?color=blue" />
+  </a>
+  <a href="https://formidable.com/open-source/urql/docs/">
+    <img alt="Spectrum badge" src="https://badgen.net/badge/docs/visit site/orange" />
   </a>
   <a href="https://spectrum.chat/urql">
-    <img alt="Spectrum badge" src="https://withspectrum.github.io/badge/badge.svg" />
+    <img alt="Spectrum badge" src="https://badgen.net/badge/spectrum/chat with us/purple" />
+  </a>
+  <a href="https://github.com/FormidableLabs/urql/blob/master/LICENSE">
```

Comment:
GitHub now displays this in the repo info header, so I think we won’t have to duplicate this 👀

Reply:
My thinking was "if you've got it, flaunt it"! I'll remove it anyhow